### PR TITLE
[SDAN-71] - Followed topic email notifications

### PR DESCRIPTION
**Pull request by akintolga**
- Initial `My Profile` page
- User level `Receive Emails` setting
- Emails will be sent for every matching story for the followed topic if this setting is on

### CODE REVIEW

File: assets/user-profile/components/UserProfileApp.jsx [hunk 1]
```diff
@@ -32,6 +38,34 @@ class UserProfileApp extends React.Component {
         }
     }
 
+    isFormValid() {
+        let valid = true;
+        let errors = {};
+
+        if (!this.props.user.first_name) {
+            errors.first_name = ['Please provide first name'];
```
Comment (petrjasek) [inline assets/user-profile/components/UserProfileApp.jsx]: gettext missing

File: assets/user-profile/components/UserProfileApp.jsx [hunk 2]
```diff
@@ -32,6 +38,34 @@ class UserProfileApp extends React.Component {
...
+            valid = false;
+        }
+
+        if (!this.props.user.last_name) {
+            errors.last_name = ['Please provide last name'];
```
Comment (petrjasek) [inline assets/user-profile/components/UserProfileApp.jsx]: same here

File: assets/user-profile/reducers.js [hunk 3]
```diff
@@ -44,6 +45,16 @@ export default function itemReducer(state = initialState, action) {
             user: action.user,
         };
     }
+
+    case EDIT_USER: {
```
Comment (petrjasek) [inline assets/user-profile/reducers.js]: I'm not sure if field changes should make it to the store, imo it's  more like internal state of the user profile component

**Comment by akintolga** [comment]
Added a new field `editedUser` for any changes that user can do

**Pull request merged**